src/DIRAC/ConfigurationSystem/Client/Helpers/Registry.py

```diff
@@ -697,3 +707,26 @@ def getEmailsForGroup(groupName):
     email = getUserOption(username, 'Email', [])
     emails.append(email)
   return emails
+
+
+def wrapIDAsDN(userID):
+  """ Wrap user ID as user DN
+
+      :param str userID: user ID
+
+      :return: str
+  """
+  return '/O=DIRAC/CN=' + userID
+
+
+def getIDFromDN(userDN):
+  """ Parse user ID from user DN
+
+      :param str userDN: user DN
+
+      :return: S_OK(str)/S_ERROR()
+  """
+  prefix = '/O=DIRAC/CN='
```

Comment:
I suggest you make the prefix a global string constant, e.g. ID_DN_PREFIX = "/O=DIRAC/CN='' to define it once

Reply:
fixed